test/controllers/declaration/DeclarantDetailsPageControllerSpec.scala

```diff
@@ -35,20 +40,37 @@ class DeclarantDetailsPageControllerSpec extends CustomExportsBaseSpec {
     withCaching[Choice](Some(Choice(Choice.AllowedChoiceValues.SupplementaryDec)), choiceId)
   }
 
+  override def afterEach() {
+    reset(mockExportsCacheService)
```

Comment:
`super.afterEach()`

otherwise if there is something defined it never gets called and you'll get all sorts of magical errors